python/lsst/ip/isr/overscan.py

```diff
@@ -89,30 +89,51 @@ def __init__(self, statControl=None, **kwargs):
             self.statControl = statControl
         else:
             self.statControl = afwMath.StatisticsControl()
+            self.statControl.setNumSigmaClip(self.config.numSigmaClip)
 
     def run(self, ampImage, overscanImage):
-        self.statControl.setNumSigmaClip(self.config.numSigmaClip)
+        """Measure and remove an overscan from an amplifier image.
 
+        Parameters
+        ----------
+        ampImage : `lsst.afw.image.Image`
+            Image data that will have the overscan removed.
+        overscanImage : `lsst.afw.image.Image`
+            Overscan data that the overscan is measured from.
+
+        Returns
+        -------
+        overscanResults : `lsst.pipe.base.Struct`
+            Result struct with components:
+            - ``imageFit`` : scalar or `lsst.afw.image.Image`
+                Value or fit subtracted from the amplifier image data.
+            - ``overscanFit`` : scalar or `lsst.afw.image.Image`
+                Value or fit subtracted from the overscan image data.
+            - ``overscanImage`` : `lsst.afw.image.Image`
```

Comment:
This is used (optionally) in the empirical variance calculation.  That happens after the trim and assembly, and I suspect saving that image segment as part of the overscan results struct was chosen instead of deciding whether to save it during the trim step.